bigtop.bom

```diff
@@ -301,7 +301,7 @@ bigtop {
       name    = 'kafka'
       rpm_pkg_suffix = "_" + bigtop.base_version.replace(".", "_")
       relNotes = 'Apache Kafka'
-      version { base = '2.8.1'; pkg = base; release = 3 }
+      version { base = '2.8.2'; pkg = base; release = 4 }
```

Comment:
@timyuer We should reset the release number to 1 if we bump the base version.

Reply:
> @timyuer We should reset the release number to 1 if we bump the base version.

Thank you for your reminder. I have updated the release version.